scripts/common.sh

```diff
@@ -23,6 +24,7 @@ do_docker_build() {
     -t $image_name:$image_tag \
     -f $MY_SCRIPT_DIR/../Dockerfile \
     --build-arg agent_version=${agent_version} \
+    $( if [ -n "$operating_system" ] ; then echo "--build-arg GOOS=$operating_system" ; fi ) \
```

Comment:
Make operating_system required and always passed as a build arg.  Defaults make the code more complicated to reason about.